packages/suite-desktop/scripts/build.js

```diff
@@ -7,10 +7,25 @@ const pkg = require('../package.json');
 
 const electronSource = path.join(__dirname, '..', 'src-electron');
 const isDev = process.env.NODE_ENV !== 'production';
+const useMock = process.env.USE_MOCK !== undefined;
 
 const gitRevision = child_process.execSync('git rev-parse HEAD').toString().trim();
 const modulePath = path.join(electronSource, 'modules');
-const modules = glob.sync(`${modulePath}/**/*.ts`).map(m => `modules/${m.replace(modulePath, '')}`);
+const modules = glob.sync(`${modulePath}/**/*.ts`).map(m => `modules${m.replace(modulePath, '')}`);
+
+const mockPath = path.join(electronSource, 'mocks');
+const mocks = {
+    'electron-updater': path.join(mockPath, 'electron-updater.ts'),
```

Comment:
That was my first intention, and I think it should be fine doing it that way for now. I didn't decide to do it to handle special cases were the import name is not the same as the file (for example: `import `mylib/esm/dist.js`). Nesting can indeed be solved by putting the mocks in directories, so I guess it should be fine to do it the way you suggested.

Reply:
Addressed in 4200d30.

Reply:
> import mylib/esm/dist.js`

Aah, I didn't see this coming. For some reason I expected that packages export all the APIs on their top level path so that we can always just `import { whatever } from 'any_package'`.

If we'll need to mock a thing that is imported from within a library the directory nesting in `mocks` can serve us. I guess that could be good enough for starters.

> Addressed in 4200d30.

🦊